site/_posts/2020-11-13-securing-flutter-oauth.md

```diff
@@ -0,0 +1,768 @@
+
+Then, we need to click "Save". When properly configured, the application details screen should look like this:
+
+{% include _image.liquid src="/assets/img/blogs/securing-flutter-app/application-configuration.png" alt="Configuring the FusionAuth application." class="img-fluid" figure=false %}
+
+We need to make sure that we register our users to the new FusionAuth application. Doing so creates a relationship between a user and the newly created application as shown in the screenshot below:
+
+{% include _image.liquid src="/assets/img/blogs/securing-flutter-app/user-registration.png" alt="Registering the user for the new application." class="img-fluid" figure=false %}
+
+If we want, we can add more users to the "Users" and register them to this application. But you can work through this tutorial with just one registered user. 
+
+If you want to test with a real device, you should also set up ngrok to point to the local FusionAuth server, so that devices can access `http://localhost:9011`. While beyond the scope of this tutorial, here's [ngrok documentation](https://ngrok.com/docs#getting-started-expose) on how to do so. 
+
+Next, we move on to setting up the Flutter app project.
+
+## Setting up the Flutter project
+
+Setting up the Flutter project is simple and easy for every OS. We can follow the instructions in the [Flutter documentation](https://flutter.dev/docs/get-started/install). But to make things easier, a quick outline of the setup steps is provided below.
+
+First, we need to clone the Flutter SDK from GitHub by running the following command in our device terminal:
+
+```shell
+cd $HOME/dev
+git clone https://github.com/flutter/flutter.git
+```
+
+Note that we need to choose an appropriate directory in which to clone the SDK. We choose `~/dev` above.
+
+Next, we need to add the flutter tool to our path by running the following command in the terminal (and if you are doing this more than once, perhaps to your `.bashrc` or `.zshrc`):
+
+```shell
+export PATH="$PATH:$HOME/dev/flutter/bin"
+```
+
+We can check if everything is configured properly by running the following command in our terminal window:
+
+```shell
+flutter doctor
+```
+
+If everything is configured properly, we will see the following result in our terminal window:
+
+{% include _image.liquid src="/assets/img/blogs/securing-flutter-app/flutter-doctor-output.png" alt="Output of the flutter doctor command." class="img-fluid" figure=false %}
+
+If you didn't see the above messages, please review the [more detailed installation instructions](https://flutter.dev/docs/get-started/install) on the Flutter site.
+
+Note that Flutter depends on iOS and/or Android development environments being installed. Doing so is beyond the scope of this tutorial, however. If you need to do that, please consult the platform installation instructions for Android or iOS as appropriate.
+
+## Creating a new Flutter app
```

Comment:
Creating a new Flutter app in FusionAuth

Reply:
This is, unfortunately, not related to fusionauth at all (it's a flutter app on the command line).